Formula/forgit.rb

```diff
@@ -0,0 +1,17 @@
+class Forgit < Formula
+  desc "Interactive git commands in the terminal"
+  homepage "https://github.com/wfxr/forgit"
+  url "https://github.com/wfxr/forgit/releases/download/23.04.0/forgit-23.04.0.tar.gz"
+  sha256 "89826d12b551829c4bce396551b1505c0604ba29130f4b8e2f842c2ca0b4c9e3"
+  license "MIT"
+
+  depends_on "fzf"
+
+  def install
+    bin.install "bin/git-forgit" => "git-forgit"
```

Comment:
This should probably be enough:
```suggestion
    bin.install "bin/git-forgit"
```

Reply:
Or, better:
```suggestion
    prefix.install "bin"
    bash_completion.install "completions/git-forgit.bash"
    pkgshare.install "forgit.plugin.zsh"
```

Reply:
@carlocab I also had to adjust `pkgshare.install "forgit.plugin.zsh"` to `prefix.install "forgit.plugin.zsh"` because the shell plugin assumes that the forgit script is located in a subdir `bin` relative to the plugin location.